content/develop/tutorials/execution-flow/fragments/create-a-multiple-container-fragment.md

```diff
@@ -0,0 +1,261 @@
+---
+title: Create a fragment across multiple containers
+slug: /develop/tutorials/execution-flow/create-a-multiple-container-fragment
+---
+
+# Create a fragment across multiple containers
+
+Streamlit lets you turn functions into [fragments](/develop/concepts/architecture/fragments), which can rerun independently from the full script. If your fragment doesn't write to outside containers, Streamlit will clear and redraw all the fragment elements with each fragment rerun. However, if your fragment _does_ write elements to outside containers, Streamlit will not clear those elements during a fragment rerun. Instead, those elements accumulate with each fragment rerun until the next full-script rerun. If you want a fragment to update multiple containers in your app, use [`st.empty()`](/develop/api-reference/layout/st.empty) containers to prevent accumulating elements.
+
+## Applied concepts
+
+- Use fragments to run two independent processes separately.
+- Distribute a fragment across multiple containers.
+
+## Prerequisites
+
+**`streamlit>=1.33.0`**
+
+- This tutorial uses fragments, which require Streamlit version 1.33.0 or later.
+- This tutorial assumes you have a clean working directory called `your-repository`.
+- You should have a basic understanding of fragments and `st.empty()`.
```

Comment:
I wonder if we should add this "you should have a basic understanding of fragments" (and maybe link to the core concept?) in all 3 tutorial pages, just in case someone lands here first.

Reply:
Updated the prerequisites in the other tutorials. All tutorials have links to the conceptual page for fragments in the first paragraph. I didn't include them here since I'm hesitant to repeat the same link close together and the general rule is to put the link on the first occurrence. Happy to revisit if it seems people are missing the link in the first paragraph and feeling a moment of awkwardness from jumping straight to the bullet points.